libexec/cli/stop-all.exec

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# 
+# Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+#
+# This software is licensed under a 3-clause BSD license.  Please
+# consult LICENSE file distributed with the sources of this project regarding
+# your rights to use or distribute this software. 
+#
+# 
+
+
+## Basic sanity
+if [ -z "$SINGULARITY_libexecdir" ]; then
+    echo "Could not identify the Singularity libexecdir."
+    exit 1
+fi
+
+## Load functions
+if [ -f "$SINGULARITY_libexecdir/singularity/functions" ]; then
+    . "$SINGULARITY_libexecdir/singularity/functions"
+else
+    echo "Error loading functions: $SINGULARITY_libexecdir/singularity/functions"
+    exit 1
+fi
+
+## Init Singularity environment
+if [ -f "$SINGULARITY_sysconfdir/singularity/init" ]; then
+    . "$SINGULARITY_sysconfdir/singularity/init"
+fi
+
+
+if ! USERID=`id --real --user`; then
+    message ERROR "Could not ascertain user ID\n"
+    exit 255
+fi
+
+
+while true; do
+    case ${1:-} in
+        -h|--help|help)
+            if [ -e "$SINGULARITY_libexecdir/singularity/cli/$SINGULARITY_COMMAND.help" ]; then
+                cat "$SINGULARITY_libexecdir/singularity/cli/$SINGULARITY_COMMAND.help"
+            else
+                message ERROR "No help exists for this command\n"
+                exit 1
+            fi
+            exit
+        ;;
+        -a|--all)
```

Comment:
hmm, since we have this shared action to parse common arguments for the daemon...
  - if it's a trivial addition, and if daemon commands are needed with others, we can add to the already existing argparser (assuming no conflict or reason not to)
 - if there are a substantial number of daemon specific commands, we could add them to a common daemon command argparser file.

The reason for this is making updates and changes. It's a lot easier to update/change one shared function than have to do the same things across multiple files.